src/Data/Nat/Properties.agda

```diff
@@ -463,6 +466,10 @@ i+j≡0⇒i≡0 (suc i) ()
 i+j≡0⇒j≡0 : ∀ i {j} → i + j ≡ 0 → j ≡ 0
 i+j≡0⇒j≡0 i {j} i+j≡0 = i+j≡0⇒i≡0 j (trans (+-comm j i) (i+j≡0))
 
+m≢0⇒m≡s[pred[m]] : ∀ {m} → m ≢ 0 → m ≡ suc (pred m)
```

Comment:
Really nitpicky I know, but usually the complex argument goes on the left hand side of the equality. Also I'd probably call it `m≢0⇒suc[pred[m]]≡m`. Maybe we can even drop `m≢0`, not sure though...